src/components/Settings/ProfileEdit.vue

```diff
@@ -93,6 +93,12 @@ export default {
       }
     },
   },
+  methods: {
+    maybeSave () {
```

Comment:
I should remove the maybeSave method from ProfileEdit.vue and should write 
`
@keyup.ctrl.enter="send"
`
instead?
By canSave do you mean maybeSave? Is that a typo?

Reply:
You add this code:

```
  methods: {
    maybeSave () {
      if (!this.canSave) return
      this.save()
    },
  },
```

`canSave` is not defined in this component, so the code will not work. But you don't need the code at all, because you can directly use `save` as you wrote: `@keyup.ctrl.enter="send"`.

`canSave` is used in other components to check if the user filled all necessary form fields and prevent saving if this is not the case.